src/SourceBuild/Arcade/tools/SourceBuildArcadeTarball.targets

```diff
@@ -53,8 +53,9 @@
       <Output TaskParameter="ConsoleOutput" ItemName="RootRepoUri" />
     </Exec>
 
+    <!-- In order to support the internal repo naming convention "dotnet-" is trimmed -->
     <Exec
-      Command="cd $(RepoRoot);git rev-parse HEAD"
+      Command="cd $(RepoRoot);git rev-parse HEAD | sed 's/^dotnet-//'"
```

Comment:
I don't think this is going to fix the problem.  In this case, the result from `git rev-parse HEAD` is a commit SHA, and doesn't have the url in  it, so the sed will be a no-op.  I think that the problem is a few lines lower, where `Tarball_WriteSourceProperties` is called.  This is called with Uri set to the `RooRepoUri`, which has the `dotnet-` prefix in it.  `Tarball_WriteSourceProperties` parses the URI to get the repo name by calling `GetDefaultRepoNameFromUrl`.  I'm not sure it needs to do that, since `SourceBuildRepoName` is already passed in in metadata.  I think the fix is to use the metadata property instead of calling `GetDefaultRepoNameFromUrl` in `Tarball_WriteSourceProperties`.

Reply:
I pushed a new change that addresses this.  Please take another look.